aws_etl_tools/s3_file.py

```diff
@@ -68,6 +72,17 @@ def download_to_temp(self):
         return destination_path
 
     @classmethod
+    def from_json_from_dict(cls, data, s3_path):
```

Comment:
since `dump` doesn't care about the type of object it gets, what about generalizing this to not be dict-specific? then we could call it something like `from_json_serialized_data`